libs/portals/admin/ids-admin/src/components/Client/AdvancedSettings.tsx

```diff
@@ -19,7 +19,7 @@ interface AdvancedSettingsProps {
   allowOfflineAccess: boolean
   requireConsent: boolean
   supportTokenExchange: boolean
-  slidingRefreshTokenLifetime: number
+  accessTokenLifetime: number
```

Comment:
~nit: I know this is not your doing but couldn't this be picked from generated code, i.e. all fields?

Reply:
Done